static-site/src/pages/whoami.jsx

```diff
@@ -12,17 +12,30 @@ const UserPage = () => {
     <Fragment>
       You&apos;re logged in as <strong>{user.username}</strong>.
       <SubText>
-        You have access to the following private Nextstrain groups, which each
+        You have access to the following Nextstrain groups, which each
         contain a collection of datasets and/or narratives:
       </SubText>
 
+      Public:
+
+      <UserGroupsList>
+        {visibleGroups.filter((group) => !group.private).map((group) => (
+          <li>
+            <a href={`/groups/${group.name}`}>{group.name}</a>
+          </li>
+        ))}
+      </UserGroupsList>
```

Comment:
This lists all public groups, i.e. the list will be the same for any logged in user. Was the intent to list just the public groups that the user is a member of?

Reply:
🤦 should have checked canary after merging... I'll open a fix before we promote to nextstrain.org.

Reply:
No worries. Thanks!

Reply:
Reverted in #779 and re-opened as #780.